integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -35,14 +37,19 @@ use crate::test_loop::utils::transactions::{
 use crate::test_loop::utils::trie_sanity::{
     check_state_shard_uid_mapping_after_resharding, TrieSanityCheck,
 };
-use crate::test_loop::utils::{get_node_data, retrieve_client_actor, LoopActionFn, ONE_NEAR, TGAS};
+use crate::test_loop::utils::{get_node_data, retrieve_client_actor, ONE_NEAR, TGAS};
 use assert_matches::assert_matches;
 use near_crypto::Signer;
 use near_parameters::{vm, RuntimeConfig, RuntimeConfigStore};
 use near_primitives::test_utils::create_user_test_signer;
 use near_primitives::transaction::SignedTransaction;
 use near_primitives::views::{FinalExecutionStatus, QueryRequest};
 
+/// Default and minimal epoch length used in resharding tests.
+const DEFAULT_EPOCH_LENGTH: u64 = 6;
+/// Increased epoch length that has to be used in some tests due to the delay caused by shard shuffling.
+const INCREASED_EPOCH_LENGTH: u64 = 8;
```

Comment:
Can you expand on it please? Is the delay causing state sync / catchup to not be finished for the next epoch? In the future, how would the developer know that they need to use the increased epoch length? What kind of error would they hit?

Reply:
fixed in 06ce60c6b2041cdb659099acde6a98c65b79a860